presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -180,6 +191,10 @@ Accessors
     Returns the 2-dimensional cartesian minimum distance (based on spatial ref)
     between two geometries in projected units.
 
+.. function:: ST_SphericalDistance(SphericalGeometry, SphericalGeometry) -> double
+
+    Returns the distance in meters between two SphericalGeometry objects. Right now only Point is the only supported input type.
```

Comment:
Perhaps, `Returns the great-circle distance in meters between two SphericalGeometry points.`